Library/Formula/node-build.rb

```diff
@@ -0,0 +1,22 @@
+class NodeBuild < Formula
+  desc "Install NodeJS versions"
+  homepage "https://github.com/OiNutter/node-build"
+  url "https://github.com/OiNutter/node-build/archive/v2.0.1.tar.gz"
+  sha256 "21d62820fb918cb4866d315f9f0c157b655d994857949f179a58756c6dde7d14"
+  head "https://github.com/OiNutter/node-build.git"
+
+  bottle :unneeded
+
+  depends_on "autoconf" => [:recommended, :run]
+  depends_on "pkg-config" => [:recommended, :run]
+  depends_on "openssl" => :recommended
+
+  def install
+    ENV["PREFIX"] = prefix
+    system "./install.sh"
+  end
+
+  test do
+    system "#{bin}/node-build", "--version"
```

Comment:
A full node compile usually runs for 10 to 15min, but we could have it
install a binary instead. I'm assuming the test server is Linux x64?
On Nov 1, 2015 12:59 PM, "Dominyk Tiller" <notifications@github.com> wrote:

> In Library/Formula/node-build.rb
> <https://github.com/Homebrew/homebrew/pull/45576#discussion_r43585280>:
>
> > +  sha256 "21d62820fb918cb4866d315f9f0c157b655d994857949f179a58756c6dde7d14"
> > +  head "https://github.com/OiNutter/node-build.git"
> > +
> > +  bottle :unneeded
> > +
> > +  depends_on "autoconf" => [:recommended, :run]
> > +  depends_on "pkg-config" => [:recommended, :run]
> > +  depends_on "openssl" => :recommended
> > +
> > +  def install
> > +    ENV["PREFIX"] = prefix
> > +    system "./install.sh"
> > +  end
> > +
> > +  test do
> > +    system "#{bin}/node-build", "--version"
>
> The testpath gets cleaned up automatically. I think the tests timeout at
> about 5 minutes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/45576/files#r43585280>.
>